models/agent_helpers.go

```diff
@@ -342,6 +342,58 @@ func ChangeAgent(q *reform.Querier, agentID string, params *ChangeCommonAgentPar
 	return row, nil
 }
 
+// PMMAgentsForNode gets pmm-agent for node.
+func PMMAgentsForNode(q *reform.Querier, nodeID string) ([]*Agent, error) {
+	return findPmmAgentsForNode(q, nodeID)
```

Comment:
There is no need to have two exactly the same functions.

Reply:
I'll fix this. It because in previous version it was used to PMMAgentsForService.